lib/ca_testing/drivers/v4/local.rb

```diff
@@ -19,11 +19,16 @@ def register
               app,
               browser: browser,
               service: service,
-              capabilities: [desired_capabilities, options]
+              capabilities: capabilities
             )
           end
         end
 
+        #The order of this is important because Safari Tech Preview set the name twice
```

Comment:
```
# The order of these capabilities is important because in the internal configuration
# for the driver; these 2 objects are merged (And both will contain a browserName)
# as such we need to ensure the browserName we manually set in `desired_capabilities`
# is retained as this is the one required by safari